<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EqualizeRasterWindowClass</class>
 <widget class="QDialog" name="EqualizeRasterWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>368</width>
    <height>305</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>368</width>
    <height>305</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>368</width>
    <height>305</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>均衡化栅格显示窗口</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>122</y>
     <width>140</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>输入栅格图层：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>189</x>
     <y>122</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="startpushButton">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>190</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>运行</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelPushButton">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>190</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>260</y>
     <width>321</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
